website/web_base.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+"""
+Base functions for web site python scripts, not really intended to be run directly
+
+Attributes
+----------
+path
+    Path to this file then shifted up one directory
+CONNECTION_STRING
+    The astrodbkit2 required string for opening the simple database with sqlite
+
+Methods
+-------
+load_db
+    Loads the simple database and returns database object
+target_exists
+    Checks a target exists in the database by querying by name, returns its inventory
+target_argparse
+    If using system arguments when calling, parses the target name
+"""
+# inbuilt libs
+import argparse
+import os
+
+# 3rd party libs
+from astrodbkit2.astrodb import Database
+
+# global vars
+path = os.path.realpath(__file__)  # path to file
+path = '/'.join(path.split('/')[:-2])  # go to 1 directory up
+CONNECTION_STRING = f'sqlite:///{path}/SIMPLE.db'  # should be in base directory
+
+
+def load_db() -> Database:
+    """
+    Loads the database
+
+    Returns
+    -------
+    _
+        The database object
+    """
+    try:
+        db = Database(CONNECTION_STRING)  # open database
+    except RuntimeError as e:  # db not created
```

Comment:
Here I've set it that I want the code to die if the database isn't present rather than trying to create one in place -- for the website the database should already exist.

Reply:
That's a good call; re-building the database could take a bit of time, especially as we add more objects. The website should rely on a binary db file/postgres server for its database rather than the underlying json.